website/source/docs/secrets/ssh/index.html.md

```diff
@@ -297,22 +297,14 @@ verified using only the public key.
 
 ### Configure a CA certificate
 
-To configure the backend, an SSH key pair must be generated to serve as the CA key:
+The first thing to do is to get Vault to generate the key pair that will be used to sign any
+SSH keys:
 
 ```text
-$ ssh-keygen -N '' -f ./ca -C 'SSH CA key pair'
-```
-
-Now that we have the SSH CA key pair, these can be used to configure the SSH CA backend:
-
-```text
-$ cat ca | vault write ssh/config/ca private_key=- public_key="$(cat ca.pub | tr -d '\n')"
+$ vault write ssh/config/ca generate_signing_key=true
```

Comment:
If `generate_signing_key` defaults to `true`, we can remove the parameter to the command here.